### PR TITLE
[REF] minor code refactor on import - Pass ProcessorObject into loadSavedMapping & use it to load the formName

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
Small change towards a larger cleanup

Before
----------------------------------------
Long list of params

After
----------------------------------------
Still a long list of params but $formName replaced by processor which can later replace all the rest.

Technical Details
----------------------------------------
After a few more steps we can eliminate most of this function & use the processor - baby step towards https://github.com/civicrm/civicrm-core/pull/15034

Comments
----------------------------------------
@seamuslee001

### CODE REVIEW

File: CRM/Contact/Import/Form/MapField.php [hunk 1]
```diff
@@ -396,12 +400,16 @@ public function buildQuickForm() {
     $formName = 'document.forms.' . $this->_name;
     //used to warn for mismatch column count or mismatch mapping
     CRM_Core_Session::singleton()->setStatus(NULL);
+    $processor = new  CRM_Import_ImportProcessor();
+    $processor->setMappingID($savedMappingID);
```
Comment (eileenmcnaughton) [inline CRM/Contact/Import/Form/MapField.php]: Not actually using mappingId & metadata yet but it seemed like the 'right sized bite' to include these sets in this pr

**Comment by eileenmcnaughton** [comment]
@seamuslee001 this is passing now - can you merge it - I'm keen to get the full processor class in (this brings in more but not all) so I can start switching to it with tests.

**Comment by colemanw** [comment]
Merging based on code review & passing tests.

**Comment by eileenmcnaughton** [comment]
thanks @colemanw

**Pull request merged**